### DISCUSSION TITLE
Hacktoberfest celebration: Blogs needed for the website (Special Prize)

### DISCUSSION THREAD

**Discussion by akshitagupta15june** [akshitagupta15june/PetMe#61] [General]
Write a blog on anything related to pet or stray animals:

for example if you have a pet in your home you can write how you feel as a pet owner, if you have helped any stray animals what was your experience? these are just the example any blog related to stray animals and pets is acceptable.

You can choose any platform to write blog but we will be needing your content here for our section blogs.

The final part the best blog will receive a special prize, let me disclose it will be

Github Sticker bundle🥳

**Comment by piyushsahujtp** [comment]
I need to have work on this issue kindly provide more information regarding this if required a html file or css file for this type of blog along with the template . Now i'm providing my blog code link https://github.com/piyushsahujtp/blog.git
for docs file kindly check my docs file blog : https://docs.google.com/document/d/1IdXNG-51DgmNyuAVYDDwyRsj21gZdR-sk60vEBonjU4/edit?usp=sharing
